README.md

```diff
@@ -215,8 +215,8 @@ gclient sync
 # On windows, opengl, d3d12 and dawn backends are enabled by default.
 # On linux and macOS, opengl and dawn are enabled by default.
 # Enable or disable a specific platform, you can add 'enable_opengl', 'enable_d3d12', and 'enable_dawn' to gn args.
-# To build a release version, specify 'is_debug=false'.
-gn gen out/Release --args="is_debug=false"
+# To build a release version, specify 'is_debug=false dawn_enable_vulkan_validation_layers=false'.
+gn gen out/Release --args="is_debug=false dawn_enable_vulkan_validation_layers=false"

```

Comment:
We can remove `dawn_enable_vulkan_validation_layers=false` if we override it in .gn

Reply:
Done